src/frontend/src/components/public/PIAFormTabs/PIAAdditionalRisks/index.tsx

```diff
@@ -22,7 +22,9 @@ const PIAAdditionalRisks = () => {
   };
 
   const [risks, setRisks] = useState<Array<IAdditionRisk>>(
-    additionalRisksForm?.risks || [
+    (additionalRisksForm?.risks.length > 0
+      ? additionalRisksForm?.risks
+      : null) || [
       {
```

Comment:
You have conditional rendering followed by an OR ||. Doesn't look like the code will never reach this line. Please correct me if I am wrong.